docs/testing.rst

```diff
@@ -0,0 +1,82 @@
+Testing
+=======
+
+Overview
+-------------
+
+Falcon uses Codeception framework to test its functions, modules, features and
+API endpoints. Tests are integrated in development workflow using Circle CI.
+
+Every time when someone opens a new pull request on Github, Circle CI executes
+the following tasks:
+
+- Checkout code from the corresponding git branch.
+- Spin up docker containers.
+- Install Falcon and all its features.
+- Run tests.
+
+If you are interested in details of Falcon CI setup,  you can find latest Circle CI configuration file here: https://github.com/systemseed/falcon/blob/master/.circleci/config.yml
+
+Running tests on local
+----------------------
+
+First, run ``make tests-prepare`` to initialize testing framework.
+Now, you can run all Codeception tests using ``make tests-run`` command. You can
+run a specific test suite by passing its name a second parameter: ::
```

Comment:
"its name a second parameter" - shouldn't it be "its name as a second parameter"?